### DISCUSSION TITLE
Admin interface - editing view

### DISCUSSION THREAD

**Discussion by ronny35** [strapi/strapi#6092] [Legacy Topics]
Currently, during view configuration, it is not possible to select one specific field from one single component (no repeatable).

- Strapi version: Strapi v3.0.0-beta.18.7

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by robertpiosik** [comment]
I'm not sure if I understood your question correctly but you should be able to manage component's view configuration in component view, not Content-Type view. Components can be shared among many Content-types, that's why you cannon change anything in assigned Content-Type.

**Comment by lauriejim** [comment]
Hello! To do what with?

When you are in your "Configure the view" page, you will see the block for your compo.
Then there is a "Set the component layout" button.
If you click on it, you can update the layout of the compo.

**Comment by ronny35** [comment]
Hello,

Sorry for this misunderstanding.
When you click on your content-type, you obtain a list. In this view, if you configure the view by clicking on wheel, it would be interesting to select a field from a component in order to filter content-type.

**Comment by lauriejim** [comment]
Ho okay! Here is an issue for this feature request.

https://github.com/strapi/strapi/issues/4501
